### ISSUE TITLE
need to add alt text options for left and right logo options in bootstrap

### ISSUE THREAD

**Issue by ronm123** [thexerteproject/xerteonlinetoolkits#934]
The header group of options in Bootstrap doesn't have fields for alt text for each of the logo options. So if one or both logos are added they show up as accessibility issues due to missing alt text.
Need to add an alt text option for each logo and then make use of that alt text if added.
